js/README.md

```diff
@@ -46,7 +48,7 @@ schematic.track({ event: "query" });
 schematic.track({ event: "query", quantity: 10 });
 
 // Check a flag
-await schematic.checkFlag("some-flag-key");
+await schematic.checkFlag({ key: "some-flag-key" });
```

Comment:
the above form is for react. for js, you need to use an object format

Reply:
Do we throw an error on invalid input? We ought to, imo

Reply:
out of scope for this, but probably worth doing. if using this in js, you lose typescript support

Reply:
I don't think we'll have many customers using vanilla js